test/Table_Tests/src/Main.enso

```diff
@@ -8,11 +8,12 @@ import project.Helpers
 import project.In_Memory
 import project.IO
 
-main =
+main filter=Nothing =
     suite = Test.build suite_builder->
         In_Memory.Main.add_specs suite_builder
         IO.Main.add_specs suite_builder
         Formatting.Main.add_specs suite_builder
         Database.Main.add_specs suite_builder
         Helpers.Main.add_specs suite_builder
-    suite.run_with_filter
+
+    suite.run_with_filter filter
```

Comment:
> Please do Table_Tests as well.

Done in ed80614e8a, however as @Akirathan pointed out, this only _filters by group name_. A simple, standardized mechanism to turn argument of `main` function into useful filter is needed.